projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-file-view/components/index.ts

```diff
@@ -1,5 +1,6 @@
-export * from './case-file-view-folder-selector/case-file-view-folder-selector.component';
-export * from './case-file-view-folder/case-file-view-folder-document-actions/case-file-view-folder-document-actions.component';
```

Comment:
Please completely undo all formatting changes in this file. I don't think there are any substantive changes in this file, and you've changes single to double quotes, which is not the project standard